### PR TITLE
Array indentation

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | /
| Type?         | improvement
| Category?     | CO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | /
| How to test?  | /

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11695)

### CODE REVIEW

File: classes/PaymentModule.php
Comment (jolelievre) [inline classes/PaymentModule.php]: please add a line break after the comma (end of array)

File: classes/Product.php
Comment (jolelievre) [inline classes/Product.php:510]: sale here, please add a line break to end the array

File: classes/controller/AdminController.php
Comment (jolelievre) [inline classes/controller/AdminController.php]:
```suggestion
            )
        );
```

File: classes/stock/StockAvailable.php
Comment (jolelievre) [inline classes/stock/StockAvailable.php:211]: I would suggest:

```php
                     Hook::exec('actionUpdateQuantity',
                         array(
                             'id_product' => $id_product,
                             'id_product_attribute' => 0,
                             'quantity' => $product_quantity,
                             'id_shop' => $id_shop,
                         )
                     );
```
or
```php
                     Hook::exec('actionUpdateQuantity', array(
                         'id_product' => $id_product,
                         'id_product_attribute' => 0,
                         'quantity' => $product_quantity,
                         'id_shop' => $id_shop,
                     ));
```

File: controllers/admin/AdminCmsController.php
Comment (jolelievre) [inline controllers/admin/AdminCmsController.php]:
```php
                'id_employee' => (int) $this->context->employee->id,
            ));
```

File: controllers/admin/AdminEmployeesController.php
Comment (jolelievre) [inline controllers/admin/AdminEmployeesController.php:137]: I don't agree on these two ones, the array end needs to be aligned on `'list'`

File: controllers/admin/AdminEmployeesController.php
Comment (jolelievre) [inline controllers/admin/AdminEmployeesController.php:138]: same thing here, align with `'PS_BO_ALLOW_EMPLOYEE_FORM_LANG'`

File: controllers/admin/AdminThemesController.php
Comment (jolelievre) [inline controllers/admin/AdminThemesController.php:652]:
```php
            )
        );
```

File: controllers/front/GetFileController.php
Comment (jolelievre) [inline controllers/front/GetFileController.php]: please add a line break at the end of the array

File: src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/TrafficSeo/Meta/SetUpUrlType.php
Comment (jolelievre) [inline src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/TrafficSeo/Meta/SetUpUrlType.php]:
```php
            ])
```

File: tests-legacy/resources/ModulesOverrideInstallUninstallTest/AdminProductsController.php
Comment (jolelievre) [inline tests-legacy/resources/ModulesOverrideInstallUninstallTest/AdminProductsController.php]:
```php
        ));
```

File: tests-legacy/resources/ModulesOverrideInstallUninstallTest/AdminProductsController.php
Comment (jolelievre) [inline tests-legacy/resources/ModulesOverrideInstallUninstallTest/AdminProductsController.php]:
```php
        ));
```

File: tests-legacy/resources/ModulesOverrideInstallUninstallTest/AdminProductsController.php
Comment (jolelievre) [inline tests-legacy/resources/ModulesOverrideInstallUninstallTest/AdminProductsController.php]:
```php
        ));
```

File: tests-legacy/resources/ModulesOverrideInstallUninstallTest/Cart.php
Comment (jolelievre) [inline tests-legacy/resources/ModulesOverrideInstallUninstallTest/Cart.php]:
```php
        ), null, false);
```
or
```php
        ),
        null, false);
```

File: tests-legacy/resources/module/pscsx3241/override/classes/Cart.php
Comment (jolelievre) [inline tests-legacy/resources/module/pscsx3241/override/classes/Cart.php]:
```php
        ), null, false);
```
or
```php
        ),
        null, false);
```

File: tests-legacy/resources/module/pscsx32412/override/controllers/admin/AdminProductsController.php
Comment (jolelievre) [inline tests-legacy/resources/module/pscsx32412/override/controllers/admin/AdminProductsController.php]:
```php
        ));
```

File: tests-legacy/resources/module/pscsx32412/override/controllers/admin/AdminProductsController.php
Comment (jolelievre) [inline tests-legacy/resources/module/pscsx32412/override/controllers/admin/AdminProductsController.php]:
```php
        ));
```

File: tests-legacy/resources/module/pscsx32412/override/controllers/admin/AdminProductsController.php
Comment (jolelievre) [inline tests-legacy/resources/module/pscsx32412/override/controllers/admin/AdminProductsController.php]:
```php
        ));
```

**Review by jolelievre — CHANGES_REQUESTED** [review]
thanks @MathiasReker 
so much more pleasurable to read
just a few comments though, then it's perfect ;)

**Comment by MathiasReker** [comment]
@jolelievre I agree with you, but the reason why I did not fix this from the beginning was that I would introduce an update to PHP CS fixer and this PR is `'array_indentation' => true'`

Your suggestion could be fixed with other PHP CS fixer rules :)

Do you still want me to fix your suggestions or should I wait to not confuse https://github.com/PrestaShop/PrestaShop/pull/11718?

**Comment by MathiasReker** [comment]
@LouiseBonnard No change to wording :)

**Comment by jolelievre** [comment]
@MathiasReker ok for the update of php-cs-fixer but then you will have to integrate the PR #11718 with the cs fixes in the code, if you think this will fix all my comments then I can approve this PR

**Comment by MathiasReker** [comment]
@jolelievre unfortunately I will have to run the script on the root folder to fix your suggestions. I think that it is too many changes since this CS lacks more places. I will fix your suggestions in this PR since it is OK for you :)

File: classes/stock/StockAvailable.php
Comment (PierreRambaud) [reply classes/stock/StockAvailable.php:211]: More:
```php
Hook::exec(
    'actionUpdateQuantity',
    array(
        'id_product' => $id_product,
        'id_product_attribute' => 0,
        'quantity' => $product_quantity,
        'id_shop' => $id_shop,
    )
);
```

**Comment by jolelievre** [comment]
I'm not sure of what you suggested in the end, "this" is confusing :P
do you fix the comments in this PR (#11695) or in the next one with the config (#11718)?

**Comment by MathiasReker** [comment]
I fix them in this PR :)

**Comment by MathiasReker** [comment]
@jolelievre is it OK for now? please check. I don't want to make this PR anymore bigger. I can add `);` (still missing some places) to new line in another PR, I need some time to prepare a script for that.

EDIT: Here you go :trollface:

**Review by jolelievre — DISMISSED** [review]

**Comment by jolelievre** [comment]
thank you @MathiasReker yeah I think it's enough ^^

**Comment by jolelievre** [comment]
@PierreRambaud can you merge this one when Codacy is over please?

File: classes/Category.php
Comment (Quetzacoalt91) [inline classes/Category.php:695]: Isn't weird to see an empty line? In this example I would expect the SQL request to start immediately after the quote instead of having a carriage return

Comment (MathiasReker) [reply classes/Category.php:695]: I can fix it, but please let me do it in an other PR. I already have an open PR related to SQL requests, and I do not want to start a lot of conflicts. :)

**Review by jolelievre — DISMISSED** [review]

**Comment by jolelievre** [comment]
waiting for travis and another approval from @PrestaShop/prestashop-core-developers

**Review by mickaelandrieu — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by jolelievre** [comment]
Thank you @MathiasReker @mickaelandrieu @Quetzacoalt91

**Pull request merged**
